### PR TITLE
Fixup momoka_ergo

### PR DESCRIPTION
**Pull request by waffle87**
## Description

- Remove unnecessary defines
- Explicitly define RGB animations
- Keymap cleanup 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by keyboard-magpie — APPROVED** [review]

File: keyboards/momoka_ergo/config.h [hunk 1]
```diff
@@ -17,36 +17,28 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
- * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
- *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
- *
- */
 #define MATRIX_ROW_PINS { C6, D7, E6, B4, B5, B6, B7 }
 #define MATRIX_COL_PINS { F7, F6, F5, F4, F1, F0 }
 
 /* COL2ROW, ROW2COL */
 #define DIODE_DIRECTION COL2ROW
-#define SELECT_SOFT_SERIAL_SPEED 5
```
Comment (daskygit) [inline keyboards/momoka_ergo/config.h:41]: What makes you think this is useless? I'd be curious why they selected such a low speed.

Comment (waffle87) [reply keyboards/momoka_ergo/config.h:41]: My best guess is a misunderstanding that 5 is the fastest speed. Nonetheless, can be reverted if that's preferred.

Comment (daskygit) [reply keyboards/momoka_ergo/config.h:41]: Please, it has the potential to break their setup. (even though it likely wouldn't)

File: keyboards/momoka_ergo/config.h [hunk 2]
```diff
@@ -17,36 +17,30 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #pragma once
```
Comment (zvecr) [inline keyboards/momoka_ergo/config.h:19]: This should be put back
```suggestion

#include "config_common.h"

```

**Review by fauxpark — APPROVED** [review]

**Pull request merged**